client/src/main/java/com/paypal/selion/reports/runtime/SeLionReporter.java

```diff
@@ -37,10 +39,9 @@
     protected static DataSaver saver = null;
     protected String baseFileName = UUID.randomUUID().toString();
 
-    protected WebDriver driver;
```

Comment:
FYI this change was not here when initial peer review was done.  Noticed it and decided that I would rather have this class not hold a reference to driver when we have static getter method already for this.